### ISSUE TITLE
Add contents of text like files to search engine

### ISSUE THREAD

**Issue by johnf** [paradisec-archive/nabu#757]
Now that we've switch to opensearch we should start indexing the contents of text like files/essences e.g. elan, pdf, rtf, doc and add these to the search engine.

**Comment by nthieberger** [comment]
Does that mean a seach within NABU would also find text within files? That is the virtue of the ROCrate solution so it is intersting if it could also available in NABU .... but maybe it iss best left to the new version if it wil take effort to include in NABU?

**Comment by johnf** [comment]
Yes, my thinking is that whenever an item that can be converted to text is uploaded, we add it to Elasticsearch and make the text searchable. I suspect I'd add it as part of the ingest pipeline.

Will need some thinking on the proper workflow and how to get it right but wouldn't be particularly difficult

Given that we are probably only talking about gigabytes of text, it might even make sense to add it to the database to make reindexing trivial.

**Comment by johnf** [comment]
Probably not something to work on straight away but I created some new issues as I was cleaning up GitHub yesterday

**Comment by johnf** [comment]
Done
